projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/vdk_ui.py

```diff
@@ -143,37 +143,36 @@ def create_job(name: str, team: str, path: str, local: bool, cloud: bool):
         return f"Job with name {name} was created."
 
     @staticmethod
-    def create_deployment(name: str, team: str, path: str, reason: str, enabled: bool):
+    def create_deployment(name: str, team: str, path: str, reason: str):
         """
         Execute `Deploy job`.
         :param name: the name of the data job that will be deployed
         :param team: the team of the data job that will be deployed
         :param path: the path to the job's directory
         :param reason: the reason of deployment
-        :param enabled: flag whether the job is enabled (that will basically un-pause the job)
         :return: output string of the operation
         """
+        import io
+        import sys
+
+        buffer = io.StringIO()
+        sys.stdout = buffer
```

Comment:
Why is the redirect necesary. It's not very clearly explained in the PR description. And I need to understand that before I can review if this solution is would be suitable.

Reply:
this is no more actual with the latest changes, I will resolve this conversation